monitoring/accessing-third-party-service-apis.adoc

```diff
@@ -0,0 +1,23 @@
+[id="accessing-third-party-service-apis"]
+= Accessing third-party Service APIs
+include::modules/common-attributes.adoc[]
+:context: accessing-third-party-service-apis
+
+toc::[]
+
+{product-title} provides direct access to third-party web service APIs for components in the Monitoring stack, such as Prometheus, Alertmanager, or Thanos Querier, using a Bearer Token for authentification.
```

Comment:
normal users with their  Bearer Token won't be able to get the alertmanager-main route, should clarify the least required permission